packages/relay/src/utils/TransactionHelper.ts

```diff
@@ -9,15 +9,15 @@ export async function addActionCount(
 ): Promise<void> {
     await db
         .transaction(async (txDB) => {
-            action(txDB)
+            const actionCount = action(txDB)
 
             const counter = await db.findOne('EpochKeyAction', {
                 where: {
                     epochKey: epochKey,
                 },
             })
 
-            const count = counter ? counter.count + 1 : 1
+            const count = counter ? counter.count + actionCount : 1
```

Comment:
Should this be `const count = counter ? counter.count + actionCount : actionCount`?

Reply:
if there's no counter data, then it will insert this data into `EpochKeyAction` which means the count should be 1.
But, `const count = counter ? counter.count + actionCount : actionCount` is more readable, I just change the code and add one more condition if the count is 0, delete this data.